CONTRIBUTING.md

```diff
@@ -57,6 +57,9 @@ python -m pytest tests/<test_file_name>.py
 
 
 
+# Code Formatting
+In order to keep our codebase clean and readible, we are using PEP8 guidelines. To help us maintain and check code style, we are using [black](https://github.com/psf/black). Simply run `black --line-length 105 .` before commiting. Failure to do so may fail the tests run on Travis. This package should have been installed for you. 
```

Comment:
Nit:  but we aren't using Travis anymore right? Also I think it's just `black .`?

Reply:
I fixed this in the CONTRBUTING doc.